src/core/Akka.Docs.Tests/Actors/ReceiveTimeoutSpecs.cs

```diff
@@ -30,40 +30,38 @@ public class ReceiveTimeoutSpecs : TestKit
         public class CheckTimeout : INotInfluenceReceiveTimeout { }
         public class ReceiveTimeoutActor : ReceiveActor
         {
-            private bool _timedOut = false;
+            private readonly TimeSpan _inactivityTimeout;
```

Comment:
Nevermind, I will fix this racy spec in separate PR: https://github.com/akkadotnet/akka.net/pull/4775